README.md

```diff
@@ -58,11 +58,15 @@ at `window.videojs` and will throw an error if it does not exist.
   If you are using jQuery, this polyfill must be loaded and initialized before jQuery is
   initialized. Unfortunately, the Chromecast framework loads the `webcomponents.js`
   polyfill via a dynamically created `<script>` tag. This causes a race condition (see
-  #17). Setting `preloadWebComponents` to `true` will make this plugin add the
-  `webcomponents.js` polyfill synchronously when the polyfill is needed. If you use the
-  `preloadWebComponents: true` option, you should make sure that this plugin is loaded
-  before jQuery. Then include the Chromecast framework after this plugin as you normally
-  would.
+  #17). Also, including `webcomponents.js` anywhere on the page will break jQuery's fix for
+  bubbling some events to `document` (e.g. `onchange` events for `<select>`, see #21).
+  Setting `preloadWebComponents` to `true` will fix these 2 problems by (1) making
+  this plugin add the `webcomponents-lite.js` polyfill synchronously when the
+  polyfill is needed and (2) using the webcomponents-lite.js version as it does
```

Comment:
Can you add code ticks (i.e. \`webcomponents-lite.js\`) around `webcomponents-lite.js`? Please forgive my formatting neurosis.

Sorry for not clarifying this earlier, but it would be nice to distinguish that the first fix is including any version of the webcomponents polyfill (unfortunately named with the `.js`, `webcomponents.js`) synchronously, and the second fix is only using the lite version of the polyfill. Maybe change `webcomponents-lite.js` in the first fix to "webcomponents"?

Reply:
Fixed the formatting and changed `webcomponents-lite.js` to `webcomponents`.

FYI, just loading `webcomponents-lite.js` before the Chromecast framework is loaded should fix both issues. The original issue with the race condition does not exists when using the "lite" version since it doesn't wrap the DOM elements. However, we need to load it synchronously because it needs to create `document.registerElement()` before the Chromecast framework checks for its existence, otherwise the framework will still load the full version of the polyfill.

Reply:
Ah, good point. Can you note your FYI as a comment in #21? That is valuable information to hold on to and with the changes GitHub makes to the PR review interface, I fear that this might get lost.